src/gluonts/model/n_beats/_ensemble.py

```diff
@@ -296,8 +296,8 @@ class NBEATSEnsembleEstimator(Estimator):
         If type is "T" (trend), then it corresponds to the degree of the polynomial.
         If the type is "S" (seasonal) then its not used.
         A list of ints of length 1 or 'num_stacks'.
-        Default and recommended value for generic mode: [2]
-        Recommended value for interpretable mode: [2]
+        Default value for generic mode: [32]
+        Recommended value for interpretable mode: [3]
```

Comment:
Yes, my suggestion was to also do this change

```suggestion
        Default and recommended value for interpretable mode: [3]
```